open-craft/opencraft
# [BB-2238] Add Footer customization page

**Styling of the legal links needs some tweaks.**

This PR adds footer customization page to the new Ocim frontend.

**Testing instructions:**
1. Checkout this branch.
1. Run Ocim (inside vagrant with `make run.dev`).
1. Go to `frontend/` and build the new API client with `./scripts/build-api-client.sh`
1. Run the new frontend: `npm start`.
1. Create a new test instance and log in the console.
1. Go to the `Theme` -> `Footer` page and make sure it's working as expected (http://localhost:3000/console/theming/footer).
1. Change colors and check if the values are correctly saved in the backend.

**Reviewers:**
